omod/src/main/webapp/resources/htmlforms/htn_dm_visit.xml

```diff
@@ -163,7 +163,7 @@
 
                     <td>
                         <!--  Pulse rate -->
-                        <obs conceptId="$pulse"/>
+                        <obsreference conceptId="$pulse"/>
```

Comment:
I noticed that "pulse" was on this form as well so I added reference tag there.

Reply:
Thanks @mogoodrich !